xds/googledirectpath/googlec2p.go

```diff
@@ -191,5 +191,8 @@ func newNode(zone string, ipv6Capable bool) *v3corepb.Node {
 // direct path is enabled if this client is running on GCE, and the normal xDS
 // is not used (bootstrap env vars are not set).
```

Comment:
Nit: Can you please update this comment.

Reply:
done